js/app.js

```diff
@@ -2,6 +2,10 @@
 
 var navView = {};
 
+// CLEARING PAGE OF CONTENT EXCEPT SKILLS (EQUATE TO HOME)
+$('.category').not('#skills').hide();
```

Comment:
I like the use of the .not() method here. So this hides all of the id.category elements, except for the skills one.